app/models/auth.py

```diff
@@ -0,0 +1,55 @@
+from app.models.db_conn import DatabaseConnection
```

Comment:
This seems out of scope since we're dealing with UI/UX design. I don't expect to see `backend` changes in this PR

Reply:
ok i will pay attention to it